### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by earth2travis**
Update Discord and .fun links

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ const Footer = () => {
             </li>
             <li className="mb-4">
               <a
-                href="https://moloch.daohaus.fun"
+                href="https://moloch.daohaus.club"
```
Comment (skuhlmann) [inline src/components/Footer.tsx]: This one doesn't seem like it should be .club

this is an old pokemol deployment:
https://moloch.daohaus.club/

Probably want:
https://moloch.daohaus.fun/

File: src/components/MolochRises.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ const MolochRises = () => {
             </div>
           </div>
         </Card>
-        <a href="https://moloch.daohaus.fun" target="_blank" rel="noreferrer">
+        <a href="https://moloch.daohaus.club" target="_blank" rel="noreferrer">
```
Comment (skuhlmann) [inline src/components/MolochRises.tsx]: .fun for this one too

File: src/components/Testimonials.tsx [hunk 3]
```diff
@@ -7,21 +7,21 @@ const testimonialData = [
...
-    href: 'https://admin.daohaus.fun/',
+    href: 'https://admin.daohaus.club/',
   },
   {
     title: 'Raid Brood',
     description:
       'NFTs are rewards for purchasing beer and then staked for membership into a DAO that collectively owns products.',
     buttonText: 'Read More',
-    href: 'https://hub.daohaus.fun/',
+    href: 'https://hub.daohaus.club/',
```
Comment (skuhlmann) [inline src/components/Testimonials.tsx]: This is a dead link. Probably some placeholder?

Maybe use https://brood.raidguild.org/

File: src/components/Testimonials.tsx [hunk 4]
```diff
@@ -7,21 +7,21 @@ const testimonialData = [
     description:
       'Consisting of core Ethereum developers, Protocol Guild prioritizes minimal governance in member engagement.',
     buttonText: 'Read More',
-    href: 'https://summon.daohaus.fun/',
+    href: 'https://summon.daohaus.club/',
```
Comment (skuhlmann) [inline src/components/Testimonials.tsx]: These link don't make a ton of sense!

**Review by skuhlmann — CHANGES_REQUESTED** [review]
Mostly all good. Just fix the moloch.daohaus.club back to .fun

**Comment by earth2travis** [comment]
@skuhlmann good catches. Made the updates here [fix(site): update Moloch and testimonial links](https://github.com/HausDAO/website/pull/38/commits/f2af11a7c514d73a0a28e5758891824222b4d103)

**Review by skuhlmann — APPROVED** [review]

**Pull request merged**